tokio-postgres/src/error/sqlstate.rs

```diff
@@ -1498,6 +1503,7 @@ static SQLSTATE_MAP: phf::Map<&'static str, SqlState> =
         ("55006", SqlState::OBJECT_IN_USE),
         ("42P01", SqlState::UNDEFINED_TABLE),
         ("25P03", SqlState::IDLE_IN_TRANSACTION_SESSION_TIMEOUT),
+        ("57P05", SqlState::IDLE_SESSION_TIMEOUT),
```

Comment:
It didn't seem like there was any specific order here, so I just put this next to a similar timeout error. Let me know if I missed something.

Reply:
This file is generated by the `codegen` crate in this repo. The proper way to update it is to update errcodes.txt, pg_range.dat, and pg_type.dat and `cargo run` the codegen.

Reply:
Oops, totally missed the comment at the top. I'll do that now.

Reply:
It looks like there have been quite a few changes since this was last updated. Specifically multi-ranges causes the type gen code to panic becuase it thinks the oid is for than normal range type. I can take a look and see how we can update the code for multi-ranges. Let me know if you have any thoughts/ideas.